addons/account/models/account_move_line.py

```diff
@@ -2973,3 +2973,16 @@ def _get_downpayment_lines(self):
         This method is overridden in the sale order module.
         '''
         return self.env['account.move.line']
+
+    # -------------------------------------------------------------------------
+    # PLACEHOLDERS
+    # -------------------------------------------------------------------------
+
+    def find_fit_product(self):
+        return self.env['product.product']
+
+    def find_fit_account(self):
+        return self.env['account.account']
+
+    def find_fit_taxes(self):
+        return self.env['account.tax']
```

Comment:
Those functions should be private. 

Also, Each of them needs a docstring explaining what they are supposed to do, and basically explain they are overridden in enterprise by account_accountant.

I don't like the idea of having _predict_* functions in account_accountant AND these ones (as I said on the enterprise PR). To me, we should do everything in _predict_ functions; so they should be the ones declared in community.